### ISSUE TITLE
Rspec output should be configurable

### ISSUE THREAD

**Issue by tolleiv** [test-kitchen/busser-serverspec#9]
Hi,

especially when using test-kitchen / busser within a CI system it's quite odd that the test result output is just part of the stdout stream. It would be great if you could allow to pass other formatter options to rspec / serverspec. So that HTML or junit reports could be generated from the testruns.

Cheers

**Comment by ytsarev** [comment]
https://github.com/test-kitchen/test-kitchen/pull/783

Here I implemented generic way to get a file from an instance to host.

**Comment by atward** [comment]
+1 for this. Though the solution should really be provided upstream in test-kitchen/busser, as it stand the format options are implemented in each busser plugin.

For anyone using TeamCity, CI integration works using [rspec-teamcity](https://github.com/apechimp/rspec-teamcity).
```ruby
# test/integration/helpers/serverspec/Gemfile
gem 'rspec-teamcity'
```
```ruby
# test/integration/helpers/serverspec/spec_helper.rb
require 'rspec/teamcity'
RSpec.configure do |config|
  config.add_formatter Spec::Runner::Formatter::TeamcityFormatter
end
```
Output isn't pretty as it still includes color + documentation formatter passed in by busser-serverspec, but results are extracted as expected.

**Comment by cl-lab-k** [comment]
you can use `shell verifer` instead of `busser`, it lets `serverspec` run with any options.

* https://github.com/test-kitchen/test-kitchen/pull/741 (it has been merged TK 1.5)
* https://rubygems.org/gems/kitchen-verifier-shell (if you use TK 1.4, install gem)

here is my `shell verifer` and `serverspec` example:
https://github.com/cl-lab-k/apache2-take/tree/shell_verifier

I have no plan to make any drastic changes to `busser-serverspec`.
And I think `shell-verifier` + `serverspec` is better choice for testing cookbooks than `busser` verifier + `busser-serverspec`.
So I am preparing the document for translation from`busser` verifier + `busser-serverspec` to `shell-verifier` + `serverspec`.
If the document is prepared, I will put link to it in README.

**Comment by cl-lab-k** [comment]
https://github.com/test-kitchen/busser-serverspec#-notice
